alchemist-incarnation-collektive/src/main/kotlin/it/unibo/collektive/alchemist/device/sensors/LocalSensing.kt

```diff
@@ -0,0 +1,11 @@
+package it.unibo.collektive.alchemist.device.sensors
+
+/**
+ * Distance sensor.
+ */
+interface LocalSensing {
+    /**
+     * Sense a molecule given its [name] and returns its value.
```

Comment:
What happens if the `name` does not match any molecule? `null` or an exception?
Please, document the case when `name` does not match any molecule.